tree/ntuple/v7/inc/ROOT/RNTupleZip.hxx

```diff
@@ -137,8 +136,15 @@ public:
    }
 };
 
-} // namespace Internal
+namespace Experimental {
+// TODO(gparolini): remove before branching ROOT v6.36
+using RNTupleCompressor [[deprecated("ROOT::Experimental::RNTupleCompressor moved to ROOT::RNTupleCompressor")]] =
+   ROOT::Internal::RNTupleCompressor;
+using RNTupleDecompressor [[deprecated("ROOT::Experimental::RNTupleDecompressor moved to ROOT::RNTupleDecompressor")]] =
+   ROOT::Internal::RNTupleDecompressor;
 } // namespace Experimental
```

Comment:
Given it's `Internal` classes, do we actually need the `using` declarations? (if so, the message needs to be updated)